lib/testing/gpuBeamformSimulate.cpp

```diff
@@ -556,13 +556,13 @@ void gpuBeamformSimulate::main_thread() {
                                                                + 1];
                                 out_sq += tmp_real * tmp_real + tmp_imag * tmp_imag;
                             } // end for ff
-                        }     // end for tt
-                    }         // end for pol
+                        } // end for tt
+                    } // end for pol
                     cpu_final_output[out_id] = out_sq / 48. / BP[int((f + 8) % 16)];
 
                 } // end for freq
-            }     // end for time
-        }         // end for beam
```

Comment:
I don't love this change to the linter, but we can live with it...  Although would be mildly tempted to remove the commend spacing rule, sometimes it's nice to line up comments like this.

Reply:
Yeah, I don't like it either. This looks to be enforced by the `AlignTrailingComments: true` option, which I'm not sure we can turn off without seeing poor formatting elsewhere.